### ISSUE TITLE
Extremely low training loss

### ISSUE THREAD

**Issue by MyElliot** [MLI-lab/DeepDeWedge#17]
Hi,
    Recently we tried DeepDeWedge with EMPIAR_11830, but we found the training loss was extremely low, in a scale of 1E-10. Although the training loss reduced as the num. of epoch grew (still in 1E-10 scale), the DDW denoised tomograms with model after 250 epochs and the full-frame tomograms seemed identical. Different random seeds were used, yet no change on this issue.  Additionally, everything is OK if we train a cryoCARE model with our data. 
    Could you please provide some suggestions on it? Thank you!
![Screenshot from 2024-10-24 11-08-43](https://github.com/user-attachments/assets/703d2e0d-6786-4232-ace3-2ad376aef8bd)

Best,
Weiping

**Comment by SimWdm** [comment]
Hi Weiping,

thanks for trying DeepDeWedge and for reaching out!
The low training losses indeed seem strange to me. Interestingly, the initial validation loss is much larger (0.178). 🤔 
Do you have the training and validation curve for this run?

Best,
Simon

**Comment by MyElliot** [comment]
Hi Simon,

Sure! Here is the curve. Finally after 2 days of training, it reaches over 700 epochs, suprisingly looks fine and converges. So I try the model after 700 epochs of training, and it refine the tomos well with increased SNR and filled missing wedge.  The low losses seem not to affect the training.
Considering the time consumption, I think an update of multi-GPU acceleration version of DeepDeWedge would indeed help a lot!

Best,
Weiping

![version_0_train](https://github.com/user-attachments/assets/be5cbac4-ba99-4c01-9e98-2bf0f51c23ba)
![Screenshot from 2024-10-26 00-46-08](https://github.com/user-attachments/assets/cc73c241-0197-4d87-862d-530c7a0d52a6)

**Comment by SimWdm** [comment]
That's great news!

I am currently working on multi GPU training. Please follow [this issue](https://github.com/MLI-lab/DeepDeWedge/issues/15) to stay up-to-date.

Other things you could try to speed up model fitting is using a smaller U-Net with fewer channels. You could try `chans: 32` instead of `chaos: 64` which is used in the demo `config.yaml`. 

Another possibility would be to increase the learning rate. Here, my suggestion would be to try for example `lr: 0.004` instead of `lr: 0.0004`.

**Comment by MyElliot** [comment]
Thank you for your suggestions and contribution, Simon! Looking forward to the update!

**Comment by SimWdm** [comment]
Hi @MyElliot, I have just pushed a multi GPU version to a new branch (which I will merge once it has been tested enough).
For more details, see [this comment](https://github.com/MLI-lab/DeepDeWedge/issues/15#issuecomment-2456761713) 😊
